### PR TITLE
Feature/automatic create changelog from commits

### PR DESCRIPTION
**Pull request by W3-Anjan**
## Description

1. Release tag & Change Log automatically from commits
2. npm environment & package.json file added
3. All required changes were made on the repo
4. Semantic versioning will be automatically maintained.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    "hooks": {
+      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
+    }
+  },
+  "commitlint": {
+    "extends": [
+      "@commitlint/config-conventional"
+    ]
+  },
+  "repository": {
```
Comment (sabbir360) [inline package.json:30]: Why its using personal repo?
@W3-Anjan

File: package.json [hunk 2]
```diff
@@ -0,0 +1,72 @@
+{
```
Comment (sabbir360) [inline package.json:1]: What are the significance of adding npm related stuff in android project?
@W3-Anjan

**Review by sabbir360 — COMMENTED** [review]
Please check my comments, if its satisfied, then we can proceed further.

File: package.json (hunk 1 above)
Comment (W3-Anjan) [reply package.json:30]: package.json was created with npm init on the telemesh repository. That time I took the personal credential during the configuration. Now I have removed it.

File: package.json (hunk 2 above)
Comment (W3-Anjan) [reply package.json:1]: For automatic semantic versioning and changelog creation I have tried with several ways from internet. There some tutorial of javascript also but [ this reference](https://www.mokkapps.de/blog/how-to-automatically-generate-a-helpful-changelog-from-your-git-commit-messages/) works well. And already with this tutorial automated change log, release tag with semantic versioning is working @sabbir360 vai

**Review by sabbir360 — APPROVED** [review]

**Review by sabbir360 — APPROVED** [review]

**Review by sabbir360 — APPROVED** [review]

**Review by sabbir360 — APPROVED** [review]

**Review by sabbir360 — APPROVED** [review]

**Review by sabbir360 — APPROVED** [review]

**Pull request merged**